### PR TITLE
add pt_BR .po file

### LINKED ISSUE #4362: Offer translation to strings collected by makemessages
## Description

The newly created projects could have the translation to strings by default. 

## Rationale

I frequently create projects supporting multiple languages, and there's always the need to translate these strings manually. 

I'd happily provide the PT-BR (Brazillian Portuguese) translation if there is any interest in the proposal. I guess the user could choose which languages to include while creating a new project.

### PR DESCRIPTION
**Pull request by matheusjardimb**
## Description

Adding translations for the default messages present in any brand-new project.

Checklist:

- [x] I've made sure that tests are updated accordingly (especially if adding or updating a template option)
- [x] I've updated the documentation or confirm that my change doesn't require any updates

## Rationale

Adding Brazilian Portuguese strings: Fix #4362.

### CODE REVIEW

File: {{cookiecutter.project_slug}}/locale/pt_BR/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+
+#: {{cookiecutter.project_slug}}/templates/account/password_reset.html:16
+msgid ""
+"Forgotten your password? Enter your e-mail address below, and we'll send you "
+"an e-mail allowing you to reset it."
+msgstr "Esqueceu sua senha? Digite seu endereço de e-mail abaixo e enviaremos um e-mail permitindo que você o redefina."
+
+#: {{cookiecutter.project_slug}}/templates/account/password_reset.html:21
+msgid "Reset My Password"
+msgstr "Resetar minha senha"
```
Comment (luzfcb) [inline {{cookiecutter.project_slug}}/locale/pt_BR/LC_MESSAGES/django.po:186]: All incidences of the word `reset` must be translated by the verb `redefinir` and its other Portuguese conjugations.

```suggestion
msgstr "Redefinir minha senha"
```

**Review by luzfcb — APPROVED** [review]
Anyway, with the exception of a single word that is not translated correctly, the rest of the PT-BR translation all looks good to me.

File: {{cookiecutter.project_slug}}/locale/pt_BR/LC_MESSAGES/django.po (hunk 1 above)
Comment (matheusjardimb) [reply {{cookiecutter.project_slug}}/locale/pt_BR/LC_MESSAGES/django.po:186]: Good one, @luzfcb!

**Review by browniebroke — APPROVED** [review]

**Comment by browniebroke** [comment]
Thanks 🎉

**Pull request merged**